components/script/dom/htmlcollection.rs

```diff
@@ -221,7 +221,7 @@ impl HTMLCollection {
     pub fn elements_iter_after<'a>(&'a self, after: &'a Node) -> impl Iterator<Item=Root<Element>> + 'a {
         // Iterate forwards from a node.
         HTMLCollectionElementsIter {
-            node_iter: after.following_nodes(&self.root),
+            node_iter: after.following_nodes::<Node>(&self.root),
```

Comment:
This type annotation is useless, given `node_iter` must be an iterator that yields `Root<Node>` values.